### PR TITLE
Bookstore Milestone2: Add Redux To Project And Setup Store, Actions and Reducers.

### PR DESCRIPTION
**Pull request by carlylechia**
In this branch, I

- [x] installed React Redux
- [x] used a "feature folder" approach to structure my files
- [x] used the [ducks pattern](https://github.com/erikras/ducks-modular-redux) to write my Redux files
- [x] Completed the milestone according to the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md)

### CODE REVIEW

File: src/Redux/Books/Books.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const ADD_BOOK = 'BOOKSTORE/Books/ADD_BOOK';
+const REMOVE_BOOK = 'BOOKSTORE/Books/REMOVE_BOOK';
```
Comment (Whoistolu) [inline src/Redux/Books/Books.js:2]: Great job following the [Duck Pattern](https://github.com/erikras/ducks-modular-redux) 👍🏽

File: src/Redux/Categories/Categories.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const BOOK_STATUS = 'BOOKSTORE/Categories/Book_STATUS';
```
Comment (Whoistolu) [inline src/Redux/Categories/Categories.js:1]: Great job following the [Duck Pattern](https://github.com/erikras/ducks-modular-redux) 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @carlylechia,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Redux/Categories/Categories.js (hunk 2 above)
Comment (carlylechia) [reply src/Redux/Categories/Categories.js:1]: Thanks, man 😃

**Pull request merged**